src/lib/common/tag.h

```diff
@@ -30,6 +30,22 @@
 #include "common/Format.h"
 
 
+// Macros to be used for JSON rendering
+
+#define JSON_PROP(name) std::string("\"" + std::string(name) + "\":")
```

Comment:
It looks very funny, not closing the first std::string ...
This should work as well;

    #define JSON_PROP(name) std::string("\"") + name + "\":"